manifests/h/Helm/Helm/3.10.1/Helm.Helm.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS0.CRLF.7-2-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Helm.Helm
+PackageVersion: 3.10.1
+PackageLocale: en-US
+Publisher: Helm
+PublisherUrl: https://helm.sh/
+PublisherSupportUrl: https://github.com/helm/community
+PrivacyUrl: https://github.com/helm/community
+Author: Helm
+PackageName: Helm
+PackageUrl: https://helm.sh/
+License: Apache License 2.0
```

```suggestion
License: Apache-2.0
```
Use SPDX Short identifier https://spdx.org/licenses/Apache-2.0.html